script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9_2.txt

```diff
@@ -1,53 +1,53 @@
 <Page1>
-It has to be sliding.
+It has to be gliding.//Sliding -> falling makes no sense, but gliding -> falling does 滑走だ。
 No, the correct term is closer to falling.
-...But can I do it? It's up to fate whether I can make it there or not.
+...But can I do it? It's up to fate whether I'll make it there or not.
 <Page2>
-There's no way that it will end up being safe. Even if everything goes well, the impact of the landing will tear my muscles and break my bones.
+It's not safe by any stretch of the imagination. Even if everything goes well, the impact of the landing will tear my muscles and break my bones.
 Will I really be able to endure that pain?
 <Page3>
-I stop the projectiles in their tracks as manifestations of my worries.
+I stop the projectiles just as they are on the brink of poking my worries directly.//I think the idea here is that they're about to enter his grey matter and poke his thoughts? 俺の 迷いを 突くかのような 弾丸を 寸前で 止める。
```

Comment:
This might be a stretch but here's my interpretation.  
Shiki was hesitating about what to do. 迷いを 突くかのような 弾丸 describes how the incoming projectiles would have broken him out of his daze if they had hit him but he managed to stop them before. That might connect to line 11 as well.

Reply:
I agree, this is how I read it too. Regardless, 迷い is more like "indecision" / "indecisiveness"

Reply:
Something like this work for the two of you then?
> I stop the projectiles that prick at my indecisiveness in the nick of time.

Reply:
Something like this work for the two of you then?
> I stop the projectiles that prick at my indecisiveness in the nick of time.